### PR TITLE
add descriptions to aliases

### PR DESCRIPTION
**Pull request by rvagg**
Added a `"description"` field so we can document what these were for, as per discussion in today's Build WG meeting / https://github.com/nodejs/build/issues/1084

Needs some items filled in and some descriptions clarified because I'm guessing a little bit.

Feel free to push a commit over my `descriptions` branch if you have updates if you'd prefer to not proxy through me.

@mhdawson could you pls check over, there's some stuff about CVE and CNA that needs filling in and clarified. release-validation-alert@ also needs a description, I'm not sure how to phrase that.
@hackygolucky could you pls check the descriptions for moderation and report and let me know if they need to be clarified
@joaocgreis I need a description for ci@, iirc it's a GitHub account but I'm not sure what account

### CODE REVIEW

File: iojs.org/aliases.json [hunk 1]
```diff
@@ -58,30 +66,40 @@
...
+    "to": [
     "reis@janeasystems.com"
   ] },
-  { "from": "rvagg", "to": "rod@vagg.org" },
-  { "from": "jbergstroem", "to": "bugs@bergstroem.nu" },
-  { "from": "ci-alert", "to": [
+  { "from": "rvagg", "description": "Used for testing only", "to": "rod@vagg.org" },
+  { "from": "jbergstroem", "description": "Used for testing only", "to": "bugs@bergstroem.nu" },
+  { "from": "ci-alert",
+    "description": "Used for alerts regarding CI system problems, some members of @nodejs/build",
```
Comment (gibfahn) [inline iojs.org/aliases.json]: Which members? Maybe:

`any members of @nodejs/build who are willing.`

File: iojs.org/aliases.json [hunk 2]
```diff
@@ -58,30 +66,40 @@
...
+    "description": "build/infra team, used for infra-level accounts with various external services",
+    "to": [
     "jbergstroem@iojs.org",
     "michael_dawson@ca.ibm.com",
     "reis@janeasystems.com",
     "rod@vagg.org"
   ] },
-  { "from": "build", "to": [
+  { "from": "build",
+    "description": "build/infra team, used for infra-level accounts with various external services (prefer accounts@ for new accounts)",
```
Comment (gibfahn) [inline iojs.org/aliases.json]: Do we need to explain what it is, can't we just say it's a deprecated alias (or equivalent) to accounts@ and leave it at that?

File: iojs.org/aliases.json [hunk 3]
```diff
@@ -58,30 +66,40 @@
...
+    "to": [
     "jbergstroem@iojs.org",
     "michael_dawson@ca.ibm.com",
     "reis@janeasystems.com",
     "rod@vagg.org"
   ] },
-  { "from": "ci", "to": [
+  { "from": "ci",
+    "description": "GitHub account (TODO: which one?)",
```
Comment (joaocgreis) [inline iojs.org/aliases.json]: Perhaps something like: "nodejs-ci GitHub account and keys"

**Comment by Trott** [comment]
Can this be rebased? I think it would be very valuable to have this information in the file. (I'm needing it right now, in fact.)

**Comment by mhdawson** [comment]
@rvagg ping ?

**Comment by mhdawson** [comment]
I'm going to close this.  At this point it will be easier to submit a new PR than merge. @rvagg let me know if that is not the right thing to do.

**Pull request closed without merging**